manifests/apache/conf.pp

```diff
@@ -33,7 +33,11 @@
   Boolean $ldap_require_group                  = $puppetboard::ldap_require_group,
   Optional[String[1]] $ldap_require_group_dn   = undef,
   Stdlib::Absolutepath $virtualenv_dir         = $puppetboard::virtualenv_dir,
-) inherits puppetboard {
+) {
+  if !defined('puppetboard') {
+    fail('The puppetboard class must be defined before puppetboard::apache::conf')
```

Comment:
If it is a hard dependency, why not just include it?

Reply:
The default values of parameters above depend on `puppetboard`.

If all those parameters where optional, that could be an issue, but in this case I think it will break the catalog in any case so we can probably drop this check :-)

Reply:
Just checked and yes, this is useless: compilation already failed because of missing parameters.